### PR TITLE
Allow IPv6 address in ManagementClient (#144)

### LINKED ISSUE #144: ManagementClient does not accept IPv6 addresses
Hi

I've found a small issue in the `urlRegex` in `ManagementClient` whereby it will not accept an IPv6 address:

When I create a `ManagamentClient`, if I supply an address to `hostUrl` in the format `2001:db8:1111::50`, it will pass the Regex but fail at the `Uri.TryCreate` on line 101:
```c#
if (!urlRegex.IsMatch(hostUrl) || !Uri.TryCreate(hostUrl, UriKind.Absolute, out var urlUri))
{
   throw new ArgumentException("hostUrl is illegal");
}
```
My solution is to change the  regex from `^(http|https):\/\/.+\w$` to `^(http|https):\/\/\[*.+\w\]*$` to allow for passing in an IPv6 address with square brackets that is supported by `Uri.TryCreate`. I'll submit a PR separately.

Cheers

James

### PR DESCRIPTION
**Pull request by euronay**
fixes #144

### CODE REVIEW

**Comment by zidad** [comment]
Hi @euronay, thanks a lot for your contribution, to get this merged quickly, would you mind adding a unit test to verify the behavior?

**Comment by euronay** [comment]
Hi @zidad. No problem - I'm just looking at adding some. 
However I'm struggling to run the integration tests, locally and I notice these have failed in the build. Is there anywhere I can look for how to run them. Do I need some particular images loaded in Docker?

**Comment by euronay** [comment]
Hi @zidad. I added some tests to cover the change I made. Let me know if you want them in a different place.

I got the integration tests running locally and they all pass, but they are still failing on appveyor?

image

**Comment by zidadrtl** [comment]
hmm yeah we need to fix the build :/

**Comment by micdenny** [comment]
Appveyor has been fixed by merging another PR that has the fix in it, so now we should see if the build can run your tests along with integration tests

**Review by micdenny — APPROVED** [review]

File: Source/EasyNetQ.Management.Client/ManagementClient.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class ManagementClient : IManagementClient
         private readonly HttpClient httpClient;
 
         private readonly Regex urlRegex =
-            new Regex(@"^(http|https):\/\/.+\w$", RegexOptions.Compiled | RegexOptions.Singleline);
+            new Regex(@"^(http|https):\/\/\[*.+\w\]*$", RegexOptions.Compiled | RegexOptions.Singleline);
```
Comment (micdenny) [inline Source/EasyNetQ.Management.Client/ManagementClient.cs:34]: `\[*` accept more than one `[`? probably better `\[?`

**Review by micdenny — CHANGES_REQUESTED** [review]

File: Source/EasyNetQ.Management.Client/ManagementClient.cs (hunk 1 above)
Comment (euronay) [reply Source/EasyNetQ.Management.Client/ManagementClient.cs]: Yes that's a very good point. Updated.

**Review by zidad — APPROVED** [review]

**Review by zidad — APPROVED** [review]

**Review by micdenny — APPROVED** [review]

**Comment by micdenny** [comment]
thank you @euronay

**Pull request merged**